hack/assets/css/playground.css

```diff
@@ -50,6 +50,11 @@ html.wf-loading .styled-select.face {
 	font-size: 16px;
 }
 
+.styled-select option {
```

Comment:
@chrissimpkins This really only applies specific styles to the individual options in the dropdown, instead of letting the browser determine the style or letting it inherit the (grey) font colour of the main dropdown.

Reply:
👍